test/EVMHost.cpp

```diff
@@ -97,34 +97,28 @@ EVMHost::EVMHost(langutil::EVMVersion _evmVersion, evmc::VM& _vm):
 	{
 		evmc::address address{};
 		address.bytes[19] = precompiledAddress;
-		// 1wei
-		m_state.accounts[address].balance.bytes[31] = 1;
+		auto account = accounts[address];
```

Comment:
```suggestion
		auto& account = accounts[address];
```
This is a bug, you have to take the reference.